CHANGELOG.md

```diff
@@ -14,11 +14,11 @@
 
 ## Compatibility Notes
 
--
+- Importing filter functions directly is deprecated, import using `filterFunctions` instead.
 
 ## New Functionality
 
--
+- Support all string, number and date related filter functions. Import `filterFunctions` and use in a filter, e. g.  `.filter(filterFunctions.startsWith(BusinessPartner.FIRST_NAME).equals(true))`.
```

Comment:
General: If I see it correctly now you have to use it like
```
import { filterFunctions } from '@sap-cloud-sdk/core/v4/';

filterFunctiontions.contains(.....)
```
Which is fine. However it would be nice to have also something like a static import in java so that you can get rid of the `filterFunctions` prefix in case you use the same function multiple times:

```
import { contains } from 'filterFunctions';
contains(.....)
```

But this is not possible anymore right?

Reply:
ah forget it after your comment above,  I just tried and `import { substring } from '@sap-cloud-sdk/core';` is still possible.

Reply:
Yes, but only for the three functions that have been there before. As far as I know there is nothing that would allow you to do what you proposed, but you can do the following:
```ts
import { filterFunctions } from '@sap-cloud-sdk/core/v4';
const { contains } = filterFunctions; 
```

Reply:
I always forget about the deconstruct. 
![image](https://user-images.githubusercontent.com/56544999/86099325-a2b87300-bab7-11ea-9bed-e42ceb312233.png)